OpenOversight/app/templates/browse.html

```diff
@@ -14,7 +14,7 @@ <h1>
       </h1>
     </div>
     <div class="text-center frontpage-leads">
-      {% for department in departments %}
+      {% for department in departments | sort(attribute="state,name") %}
```

Comment:
Sort.

Reply:
Great! I would prefer to do the sorting in the view and not the template (I was hoping to at some point also move the existing logic out of the templates)

Reply:
I've generally been told that it's a better idea to do the sorting in the HTML, presuming it only matters there and the amount of information isn't HUGE, as it's only a client concern. If it was something we needed on the server side, I could see doing it there, but I think it's fine to do it here.

All that being said, I don't have any qualms doing it one way or the other. Thumbs up this message and I can take care of it on the server side.

Reply:
Done!